src/main/java/seedu/address/model/attendance/Attendance.java

```diff
@@ -21,7 +22,8 @@
             + "Month should be 01-12"
             + "\n"
             + "Year should be a 4 digit numeric value, with the exception of 0000";
-    private static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+    private static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("dd/MM/uuuu")
```

Comment:
Cool change! Went to read up on it and TIL apparently using `uuuu` is better than using `yyyy`.